aws/resource_aws_lambda_function.go

```diff
@@ -161,6 +161,11 @@ func resourceAwsLambdaFunction() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"source_code_size": {
```

Comment:
Two things:
* This does not look configurable -- we should remove `Optional: true`
* Its not documented in the resource documentation